custom-vu/src/main/kotlin/jces1209/vu/page/DcIssuePage.kt

```diff
@@ -130,6 +140,75 @@ class DcIssuePage(
         return this;
     }
 
+    override fun isTimeSpentFormAppeared(): Boolean {
+        try {
+            driver
+                .wait(
+                    ExpectedConditions.presenceOfElementLocated(By.id("log-work-time-logged"))
+                )
+        } catch (e: Exception) {
+            return false
+        }
+        return true
+    }
```

Comment:
Code seems to say: This element might not exist, so we wait 10 seconds in case it appears.
It's a problem, because it adds 10 seconds of waiting time for cases when the element will not appear.
Can we detect it immediately? Or at least reduce the wait time?